bitnami/harbor/Chart.yaml

```diff
@@ -34,4 +34,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-harbor-registry
   - https://github.com/bitnami/bitnami-docker-harbor-registryctl
   - https://goharbor.io/
-version: 9.6.2
+version: 10.0.0
```

Comment:
Hi,

I see that it is updating a minor version, so I don't see a strong reason for bumping the major version.